src/main/java/model/item/Weapon.java

```diff
@@ -1,11 +1,42 @@
 package model.item;
 
+import javafx.beans.binding.Bindings;
+import javafx.beans.binding.IntegerBinding;
+import javafx.beans.property.IntegerProperty;
+import javafx.beans.property.SimpleIntegerProperty;
+
 public class Weapon extends Equipment {
 
-    private String initiative = "0";
-    private String dice = "";
-    private int damage = 0;
-    private int hit = 0;
+    private String initiative;
+    private String dice;
+    private final IntegerProperty damage;
+    protected final IntegerBinding damageWithWear;
+    private int hit;
+
+    public Weapon() {
+        super();
+        this.initiative = "0";
+        this.dice = "";
+        this.damage = new SimpleIntegerProperty(0);
+        this.damageWithWear = Bindings.createIntegerBinding(() -> Math.max(0, getDamage() - getWear()), damage, wear);
+        this.hit = 0;
+    }
+
+    @Override
+    protected boolean shouldBreak() {
+        return getWear() > getDamage();
+    }
```

Comment:
I think this should be the default implementation

Reply:
What do you mean with default implementation?

Reply:
Dont mind me, I am stupid